src/components/TodoBox.js

```diff
@@ -0,0 +1,71 @@
+import Clock from './Clock';
+import { useCallback, useEffect, useState } from 'react';
+import TodoInput from './TodoInput';
+import TodoLists from './TodoLists';
+import styled from 'styled-components';
+
+const Title = styled.h2`
+  margin: 0;
+  font-weight: normal;
+`;
+
+const TodoBox = () => {
+  //localStorage key설정
+  const TODOS_KEY = 'todos';
+
+  //localStorage에서 todos 가져오기
+  const [todos, setTodos] = useState(
+    () => JSON.parse(localStorage.getItem(TODOS_KEY)) || []
```

Comment:
arrow function이 꼭 필요할까요?
```suggestion
    JSON.parse(localStorage.getItem(TODOS_KEY)) || []
```

Reply:
arrow function에 대한 이해가 부족한 것 같네요.. 더 공부해보겠습니다!